tests/lax_scipy_sparse_test.py

```diff
@@ -30,6 +30,7 @@
 
 from jax.config import config
 config.parse_flags_with_absl()
+# config.update("jax_enable_x64", True)
```

Comment:
Good catch! This was probably it. Can you remove this commented out line entirely?

Reply:
Done!